app/src/common/message_channel.c

```diff
@@ -81,3 +81,11 @@ ZBUS_CHAN_DEFINE(TIME_CHAN,
 		 ZBUS_OBSERVERS(battery, environmental),
 		 ZBUS_MSG_INIT(0)
 );
+
+ZBUS_CHAN_DEFINE(ZBUS_TEST_CHAN,
```

Comment:
I think the channel can be defined just locally in the shell modul as it will not be used by other modules.